src/hardware/serial_terminal.rs

```diff
@@ -25,28 +85,128 @@ impl Write for OutputBuffer {
     }
 }
 
+impl core::fmt::Write for Context {
+    /// Write data to the serial terminal.
+    ///
+    /// # Note
+    /// The terminal uses an internal buffer. Overflows of the output buffer are silently ignored.
+    fn write_str(&mut self, s: &str) -> core::fmt::Result {
+        self.output.write_str(s)
+    }
+}
+
+fn handle_list(
+    _menu: &menu::Menu<Context>,
+    _item: &menu::Item<Context>,
+    _args: &[&str],
+    context: &mut Context,
+) {
+    writeln!(context, "Available properties:").unwrap();
+    for path in Settings::iter_paths::<heapless::String<32>>("/") {
+        let path = path.unwrap();
+        writeln!(context, "* {path}").unwrap();
```

Comment:
Let's also print the current values as well as the ones that `reset` would establish while we are at it (instead of after each `read`?). Seems convenient and easy enough.

Reply:
Fixed in 6182e1c790d6b1f1f5a8e824419c6f877bdcd4f1